packages/core/useFetch/index.test.ts

```diff
@@ -0,0 +1,24 @@
+import { useFetch } from '.'
+
+describe('useFetch', () => {
+  const urls = {
+    get: 'https://httpbin.org/get',
+    post: 'https://httpbin.org/get',
+    error: 'https://httpbin.org/status/500',
+  }
+
+  test('basic get request', async() => {
+    const { statusCode } = useFetch(urls.get)
+
+    /**
```

Comment:
Would like some input here on how to go about writing the tests for this type of problem described below